src/views/design/ScriptDetail/EditAction/index.tsx

```diff
@@ -155,6 +160,10 @@ function EditAction({ onClose, action, onEdit, state }: IPublicProps & IObserveP
                             className={classes.descriptionTextField}
                             defaultValue={description}
                             onBlur={(e) => setDescription(e.target.value)}
+                            InputProps={{
+                                readOnly: !checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC'),
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'